tests/api.test.ts

```diff
@@ -318,35 +299,46 @@ describe('request', () => {
   });
 
   it('resolves with mock response headers and function returning string', async () => {
-    fetch.mockResponseOnce(() => Promise.resolve('ok'), {
-      headers: { ding: 'dang' },
-    });
-    return expect(fetch('https://bar', {}).then((response) => response.headers.get('ding'))).resolves.toEqual('dang');
+    fetch.mockResponseOnce(() =>
+      Promise.resolve({
+        body: 'ok',
+        headers: { ding: 'dang' },
+      })
```

Comment:
Question: Why use an object here instead of the two parameters as before?  Does it no longer work the same way as previously?